dev_mode/package.json

```
@@ -20,9 +20,9 @@
     "@codemirror/language": "^6.0.0",
     "@codemirror/state": "^6.2.0",
     "@codemirror/view": "^6.9.6",
-    "@jupyter/react-components": "^0.16.6",
-    "@jupyter/web-components": "^0.16.6",
-    "@jupyter/ydoc": "^3.0.0-a3",
+    "@jupyter/react-components": "^0.17.0",
+    "@jupyter/web-components": "^0.17.0",
+    "@jupyter/ydoc": "^2.0.1",
```

You gonna have issue due to this downgrade:

```suggestion
    "@jupyter/ydoc": "^3.0.0-a3",
```